pyproject.toml

```diff
@@ -54,7 +54,15 @@ processing = [
     "tldextract",
     "trafilatura>=1.8.0",
     "tokenizers",
-    "ftfy"
+    "ftfy",
+    "stanza",
```

Comment:
I suggest to keep only necessary dependency for Korean and also only code to support Korean besides English for now. It will make it easier  to review the PR and we can easily add more Languages once the structure is agreed on.